documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -27,13 +27,13 @@ ifdef::Kubernetes[]
 On {KubernetesName} this can be done using `kubectl apply`:
 +
 [source,shell]
-kubectl apply -f examples/install/cluster-operator
+kubectl apply -f examples/install/user-operator
```

Comment:
the file is about topic-operator standalone why user-operator here?

Reply:
See? I try to fix something and fix it incorrectly :-(. I might need a vacation as well :-)

Reply:
You don't need vacation ... you deserve it! ;)